composer.json

```diff
@@ -12,7 +12,8 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "~1.0@dev",
+        "php-http/httplug": "dev-master",
```

Comment:
ah, is prefer-stable hittung us? i think ^1.0.0-alpha2 should work (the branch alias is the very last 1.0 there is, so even though alpha2 does not exist yet it should work)

btw, not sure to what `1.0.0-alpha` translates. there should always be a number after the word alpha.